src/ResultWriter/faultoutput.f90

```diff
@@ -554,21 +554,17 @@ SUBROUTINE calc_FaultOutput( DynRup_output, DISC, EQN, MESH, MaterialVal, BND, t
           IF (DynRup_output%OutputMask(5).EQ.1) THEN
 
               !add transient nucleation stress to fault output
-              IF ((EQN%FL.EQ.103) .OR. (EQN%FL.EQ.3) .OR. (EQN%FL.EQ.4)) THEN
-                 NucleationStressXYZ = MATMUL(T(1:6,1:6), EQN%NucleationStressInFaultCS(iBndGP,:,iFace))
-                 Tnuc = DISC%DynRup%t_0
-                 Gnuc = Calc_SmoothStep(time, Tnuc)
-                 NucleationStressLocalCS = MATMUL(rotmat, NucleationStressXYZ)*Gnuc
-              ELSE
-                 NucleationStressLocalCS(:) = 0d0
-              ENDIF
+              NucleationStressXYZ = MATMUL(T(1:6,1:6), EQN%InitialStressInFaultCS(iBndGP,:,iFace))
+              Tnuc = DISC%DynRup%t_0
+              Gnuc = Calc_SmoothStep(time, Tnuc)
+              NucleationStressLocalCS = MATMUL(rotmat, NucleationStressXYZ)*Gnuc
```

Comment:
hum, I m not sure this works.
stress = transient + initial stress + Gnuc* nucleation_overstress.
but not stress = transient + Gnuc* (initial stress + nucleation_overstress).
So maybe, the idea of merging initial stress and nucleation overstress does not work?

Reply:
This is how we do it: https://github.com/SeisSol/SeisSol/blob/dr/cpp/src/DynamicRupture/FrictionLaws/RateAndState.h#L146-L155, so with the Gnuc removed from this code  snippet, I think we are doing it correctly.